### PR TITLE
Route/question

### PR DESCRIPTION
**Pull request by KrittinI**

### CODE REVIEW

File: package.json
Comment (KrittinI) [inline package.json:1]: add seed for run seed

File: prisma/seed.js
Comment (KrittinI) [inline prisma/seed.js:1]: create seed for more data in database

File: src/controllers/auth-controller.js
Comment (KrittinI) [inline src/controllers/auth-controller.js:1]: edit for more input to user

File: src/controllers/question-controller.js
Comment (KrittinI) [inline src/controllers/question-controller.js:1]: edit some variable name and add some logic before update to database

File: src/middlewares/check-quesiton.js
Comment (KrittinI) [inline src/middlewares/check-quesiton.js:1]: this is middleware to check data in array of question before create question

**Review by KrittinI — COMMENTED** [review]
let check our data

**Review by Nuppasorn-Pop — APPROVED** [review]

**Pull request merged**